src/content_mapblock.cpp

```diff
@@ -172,6 +172,10 @@ static inline void getNeighborConnectingFace(v3s16 p, INodeDefManager *nodedef,
 		*neighbors |= v;
 }
 
+static int NeighbourToIndex(const v3s16 &pos) {
+	return (9 * pos.X + 3 * pos.Y + pos.Z) + 13;
```

Comment:
I've added comments to better clarify how this function should be used. I've also added a comment to make it a lambda when we move to C++11, to remind us to move it into a smaller scope so it is less likely to be used outside the function. I'm not too sure about the bitshift to be honest. (In the sense I don't know - not that I disagree).